GUIDELINES.md

```diff
@@ -0,0 +1,76 @@
+# Development Guidelines
```

Comment:
File is possibly misnamed as `GUIDELINES.md` as it is mostly the needed "how to".  You can look at other repos to see what they do, but I think `BUILD.md` is common, and there are other variations out there.

Reply:
@david-bakin-sl thanks for the input. This approach is motivated by [OZ smart contract repo](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/GUIDELINES.md) where they have a file named GUIDELINES.md. But hey `BUILD.md` sounds good to me as well! Let me know what you think!

Reply:
So over there it seems their `GUIDELINES` really contains _guidelines_: If you're going to contribute here's the coding style, here's the kind of tests and documentation you need to provide.  Here you've provided a neat "how to" cookbook of how to build the thing.  The only real "guideline" is the suggestion to use prettier.  

However! My style in code reviews is to bring up stuff and let the PR author decide.  If I really think something needs to be fixed I'll mark it "needs changes".  But in most cases all check "approve" and say "LGTM" and that really means I'm offering suggestions for your evaluation.

(That's my personal style; others here may do things differently.)

Also I comment on a bunch of things that aren't even suggestions to change anything but just letting you know what I think.

Reply:
Ah I see! Yeah I actually appreciate the input and I do read all your comments haha.

Reply:
@david-bakin-sl cool so yeah well random enough Nana and I just brought this up during our 1:1, and we both heard you on this one and agree that this document is not merely a set of guidelines but rather a comprehensive set of how-to instructions. So its primary purpose is to facilitate the onboarding process for new users, guiding them through the setup and testing of smart contracts. Therefore, I have decided to rename it to "TEST_SETUP.md".  Once again, thank you for your input, and I'm always all ears for feedback 😁😁